resource_management_mobile/src/app/auth/login/login.page.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import { LoginService } from '../login.service';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { LoginRequest, LoginResponse } from '../models/auth.model';
+import { Router } from '@angular/router';
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.page.html',
+  styleUrls: ['./login.page.scss'],
+})
+export class LoginPage implements OnInit {
+  loginForm !: FormGroup;
+  showPasswordText:boolean = false;
+  type:string = 'password';
+  onSubmit:boolean = true;
+  constructor(private loginService:LoginService,private fb: FormBuilder,private router:Router) { }
+
+  ngOnInit() {
+    this.loginForm = this.fb.group({
+      userName: ['', [Validators.required]],
+      password: ['', [Validators.required]]});
+  }
+  
+  passwordShow(){
+    this.showPasswordText =! this.showPasswordText;
+    this.type =  (this.showPasswordText == true ? 'text':'password')
+  }
+
+  submit(){
+    console.log()
```

Comment:
Remove the console before commitng the code

Reply:
console.log removed.